src/containers/Login/Login.js

```diff
@@ -48,14 +52,14 @@ class Login extends Component {
       };
       API.userLogin(data, result => {
         if (result.status === "200") {
+          localStorage.setItem("id", result.data.user.uuid);
```

Comment:
@Humnaa 

move these in session service

Reply:
i have moved all session related implementation in a separate folder